java/client/src/main/java/glide/api/models/BaseTransaction.java

```diff
@@ -4077,6 +4079,183 @@ public <ArgType> T xinfoConsumers(@NonNull ArgType key, @NonNull ArgType groupNa
         return getThis();
     }
 
+    /**
+     * Transfers ownership of pending stream entries that match the specified criteria.
+     *
+     * @implNote {@link ArgType} is limited to {@link String} or {@link GlideString}, any other type
+     *     will throw {@link IllegalArgumentException}.
+     * @see <a href ="https://valkey.io/commands/xautoclaim">valkey.io</a> for details.
+     * @param key The key of the stream.
+     * @param group The consumer group name
+     * @param consumer The group consumer.
+     * @param minIdleTime The minimum idle time for the message to be claimed.
+     * @param start Filters the claimed entries to those that have an ID equal or greater than the
+     *     specified value.
+     * @return Command Response - An array containing the following elements:
```

Comment:
```suggestion
     * @return Command Response - An <code>array</code> containing the following elements:
```
Apply in this file too